codegen/s3.rs

```diff
@@ -11013,7 +11013,7 @@ impl<'a> S3Client<'a> {
 		params.put("Action", "ListObjectVersions");
 		ListObjectVersionsRequestWriter::write_params(&mut params, "", &input);
 		request.set_params(params);
-		let result = request.sign_and_execute(try!(self.creds.get_credentials()));
+		let result = request.sign_and_execute(try!(self.creds.credentials()));
```

Comment:
This highlights the poor naming for the credentials member of the S3Client struct.  I think we have freedom to call it something like `credentialsProvider` on line 11001 above so this line would read:

`let result = request.sign_and_execute(try!(self.credentialsProvider.credentials()));`

Also applies to other clients, this was just the one that jumps out since it's Rust and not Python that makes Rust.